frontend/packages/kubevirt-plugin/src/utils/strings.ts

```diff
@@ -18,6 +18,38 @@ export const ADD_NETWORK_INTERFACE = 'Add Network Interface';
 export const ADD_SNAPSHOT = 'Take Snapshot';
 export const RESTORE_SNAPSHOT = 'Restore Snapshot';
 
+// storage ui sources descriptions
+export const UI_SOURCE_BLANK_DESC = 'Create an empty disk (PVC)';
+export const UI_SOURCE_URL_DESC =
+  'Upload content via URL (HTTP or S3 endpoint). A new persistent volume claim (PVC) will be created';
+export const UI_SOURCE_CONTAINER_DESC =
+  'Upload content from a container located in a registry accessible from the cluster. Example: kubevirt/cirros-registry-disk-demo. A new persistent volume claim (PVC) will be created';
+export const UI_SOURCE_ATTACH_CLONED_DISK_DESC =
+  'Clone a persistent volume claim (PVC) already available in the cluster';
+export const UI_SOURCE_ATTACH_DISK_DESC =
+  'Use a persistent volume claim (PVC) already available on the cluster';
+export const UI_SOURCE_IMPORT_DISK_DESC = 'TBD';
+
+// provision sources descriptions
+export const PROVISION_SOURCE_PXE_DESC =
+  'Boot an operating system from a server on a network. This will show up as a NIC on the Networking step. Requires a PXE bootable network attachment definition';
```

Comment:
The show up NIC is "Pod Networking" nic, it shows up for each provision source, maybe not worth mention it here.